src/Microsoft.TestPlatform.Common/Interfaces/Engine/TesthostProtocol/IDiscoveryManager.cs

```diff
@@ -28,6 +28,6 @@ public interface IDiscoveryManager
         /// <summary>
         /// Aborts the test discovery.
         /// </summary>
-        void Abort();
+        void Abort(ITestDiscoveryEventsHandler2 eventsHandler);
```

Comment:
Would this affect compat between vstest.console and testhost*.exe? I think irrespective it would be good to validate against older versions of Test SDK that would not understand the new messages. Also please validate that UWP works as well cause it uses its own host process.

Reply:
In TestRequestHandler we have switch case with default behaviour which will log "Invalid Message types".
When vstest.console is old, but testhost is new : means that vstest.console just will not send new message
When vstest.console is new, but testhost is old : will log "Invalid Message type."

But obviously we will ask vendor team to test all scenarious and also UWP, which you mention